ci/integration/deploy-test.yaml

```diff
@@ -0,0 +1,135 @@
+---
+apiVersion: concourse.k8s.io/v1beta1
+kind: Pipeline
+metadata:
+  labels:
+    controller-tools.k8s.io: "1.0"
+  name: deploy
```

Comment:
There [already is a pipeline](https://ci.london.verify.govsvc.uk/teams/proxy-node-integration/pipelines/deploy) whose metadata is `deploy`.  I'm unsure whether this new pipeline name needs to be distinct, or will slot into the existing `deploy` pipeline.

Reply:
You should add this job to the `deploy-pipeline.yml` file. It's going to be just another job in the same pipeline.
But maybe Concourse is smart enough to combine things from different files 🤷‍♂
@chrisfarms thoughts?